### PR TITLE
Update to adopt kernel 6.2 usage changes - do NOT merge for now

### PR DESCRIPTION
**Pull request by ruomengh**
Signed-off-by: Hao, Ruomeng <ruomeng.hao@intel.com>

### CODE REVIEW

**Review by kenplusplus — APPROVED** [review]
LGTM

File: attestation/pytdxmeasure/README.md [hunk 1]
```diff
@@ -9,6 +9,26 @@ before providing the TDX guest VM.
 
 ![](https://github.com/intel/tdx-tools/blob/main/doc/tdx_measurement.png)
 
+### Overview
+
+The `RTMR` stands for Run-time Measurement Register, recording measurement for the component participating in the booting process.
+Currently (2023.4), TDX supports four RTMRs, including RTMR[0], RTMR[1], RTMR[2] and RTMR[3].
```
Comment (hwang37) [inline attestation/pytdxmeasure/README.md:15]: Currently (2023.4) => As of 2023.4,

File: attestation/pytdxmeasure/examples/extend_ima.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    /sys/kernel/security/ima/ascii_runtime_measurements
+To enable IMA in kernel for TDs, please set the kernel configs like this:
+    CONFIG_INTEGRITY=y
+    CONFIG_IMA=y
+    CONFIG_IMA_MEASURE_PCR_IDX=10
+    CONFIG_IMA_LSM_RULES=y
+Also additional kernel command needed to be added for IMA enablement
+    ima=on, ima_policy=tcb, ima_hash=sha384
+
+Traditionally ima measurements are anchored in Trusted Platform Module (TPM),\
```
Comment (hwang37) [inline attestation/pytdxmeasure/examples/extend_ima.py:18]: ima => IMA

File: attestation/pytdxmeasure/pytdxmeasure/utility.py [hunk 3]
```diff
@@ -0,0 +1,209 @@
+'''
+Module utility provide constant variales defined in outer
```
Comment (hwang37) [inline attestation/pytdxmeasure/pytdxmeasure/utility.py:2]: provides

File: utils/td-migration/README.md [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+
+## 2. Single Host Test Steps
+
+**NOTE**: each command needs one terminal, so please prepare 5 terminal tabs firstly or use tmux.
+
+### 2.1 Create MigTD
+
+The MigTD is designed to evaluate potential migration sources and targets for adherence to the TD Migration Policy and exchange MSK(Migration session Key) from the source platform to the destination platform. Creating MigTD is an additional workflow compared with live migration.
+
+**Prerequisite**: Please install all attestation required components in advance. (not in this readme scope) Because attestation is embedded in MigTD, if pre-mig success, it means MigTD attestation pass.
```
Comment (hwang37) [inline utils/td-migration/README.md:20]: (not in this readme scope).

File: utils/td-migration/README.md [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+
+### 2.4 Live Migration
+
+Starts to transfer TD's data from source to destination.
+
+```bash
+sudo ./mig-flow.sh
+```
+
+After migration is complete, you can see the following message and use destTD normally.
```
Comment (hwang37) [inline utils/td-migration/README.md:128]: completed

**Pull request merged**